src/util/tooltip/translate.ts

```diff
@@ -55,8 +55,9 @@ export function getTooltipTranslateXY({
     return position[key];
   }
 
-  const negative = coordinate[key] - tooltipDimension - offsetTopLeft;
+  const negative = coordinate[key] - tooltipDimension - (offsetTopLeft > 0 ? offsetTopLeft : 0);
```

Comment:
I am trying to understand why is this zero. My first intuition says it should be `Math.abs(offsetTopLeft)`.

Reply:
Negative offsets with `allowEscapeViewBox=false` result in the Tooltip stopping at the minimum edge of the viewbox at the moment, at `0`. Using `Math.abs(offsetTopLeft)` here results in the Tooltip being offset from the boundary on the maximum edge by the given value. So the behaviour for min and max edge would be inconsistent with `Math.abs(offsetTopLeft)`